src/core_plugins/kibana/public/dashboard/top_nav/show_save_modal.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { DashboardSaveModal } from './save_modal';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export function showSaveModal(onSave, title, description, copyOnSave, timeRestore, showCopyOnSave) {
```

Comment:
When there are a lot of function parameters I personally prefer putting them in a destructured config or options object so it's easier to follow when passing them in (actually only since joining here and some other reviewers pointed it out to me, then I started to really like that style).
So it would just be:
```
showSaveModal({ onSave, title, description, copyOnSave, timeRestore, showCopyOnSave }) {
...
})
```
Then the caller would just have to pass them in differently too.